src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int INDEX_OF_FIGURE = 4;
+    public static final int DEFOULT_RADIUS = 10;
+
+    public static final Color DEFOULT_COLOR = Color.valueOf("WHITE");
+    public static final int MAX_VALUE = 25;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomFigure = random.nextInt(INDEX_OF_FIGURE);
+        switch (randomFigure) {
+            case 0 :

```

Comment:
it's better to switch through the figures, not indexes, to make this switch-case more understandable
consider creating enum for shape and generating random one to create this